<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>875</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="up">
        <item>
         <layout class="QVBoxLayout" name="code">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Code</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="CodeDisplay">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="result">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Result</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="ResultDisplay">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="tree">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string> Syntax Tree</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="TreeDisplay">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="btn">
        <item>
         <widget class="QPushButton" name="btnCleanCode">
          <property name="text">
           <string>Clean</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnRunCode">
          <property name="text">
           <string>Run</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnLoadCode">
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="input">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Input</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="cmdLineEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sendButton">
          <property name="text">
           <string>Send</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
